pkg/apis/pipeline/v1beta1/taskrun_validation.go

```diff
@@ -218,6 +218,11 @@ func validateDebug(db *TaskRunDebug) (errs *apis.FieldError) {
 	if db == nil || db.Breakpoints == nil {
 		return errs
 	}
+
+	if db.Breakpoints.OnFailure == "" {
+		errs = errs.Also(apis.ErrInvalidValue("onFailure breakpoint is empty, it is only allowed to be set as enabled", "breakpoints.onFailure"))
+	}
+
```

Comment:
Is a corresponding change not needed in `apis/pipeline/v1/taskrun_validation.go`?